data/korath/korath missions.txt

```diff
@@ -257,3 +257,186 @@ mission "Small Scale Delivery"
+			`	Choot'k says, "We've destroyed the ones in this system, but we need you to fly to Almach and destroy the drones there."`
+			choice
+				`	"Just blow up drones? Why do you need me for this?"`
+					goto whyme
+				`	"I'm surprised you don't want the fight."`
+				`	"Sounds like target practice to me."`
+					goto practice
+			`	"My troops need me here to command the pacification of Stormhold. I wish I could come with; it will be a glorious fight." He almost looks sad as he says that. "The largest ship we have seen since the Korath war now guards the drones."`
+				goto navy
+			label practice
+			`	Choot'k looks worried. "You may be target practice if you aren't prepared. The drones are guarded now, by the largest ship we've seen since the Korath war. You'll need to dodge it while shooting the drones."`
+				goto navy
+			label whyme
+			`	Choot'k says, "Ah, it'll take a skilled pilot to do it. The drones are guarded now, by the largest ship we have seen since the Korath war. It is not attacking anything; we think it is just guarding the drones."`
+				goto navy
+			label navy
+			`	Batu appears annoyed briefly before returning to his usual calm. "The Navy has prepared an elite strike force that is ready to take on the task. Teeneep has generously loaned us some Hai outfits to ensure our success. However, General Choot'k-"`
+			`	Choot'k interrupts. "The urgency of my call was for you to have the right of first refusal. When you return here, successful in your mission, we'll be sure to have some good Korath gear ready for you."`
+			`	Batu sighs. "The other leaders have agreed to this. However, if you would rather not do it, cancel the mission and the Navy will take care of it-" Choot'k cuts in, "-but don't expect to gain any respect for failure."`
+			choice
+				`	"Sound's like fun! I'm in."`
+					accept
+				`	"I'm not afraid, but I am busy right now."`
+					decline
+				`	"No thanks. I don't want to steal the Navy's glory."`
+					decline
+	on enter Almach
+		dialog "The Navy's sensor algorithms are working: your ship is picking up an abundance of drones."
+	on complete
+		payment 2500000
+		conversation
+			`Choot'k greets you, beside some Navy crewman in inventory management uniforms. "The Navy tells me you shut down the transmissions. Good job! As promised, we've set aside some of the best Korath gear for you to choose from.`
+			`	"Pick among these:`
+			`	"Some wonderful Korath systems that repair your hull and shields, while also storing energy. They come with two Korath anti-missile turrets and some other odds and ends.`
+			`	"I can also offer the strongest afterburners I've ever heard of, plus a sizable amount of fuel storage and ramscoop to keep it working.`
+			`	"We can give you a power generator that's as big as two fighters, and a lot of cooling outfits to support it.`
```

Comment:
```suggestion
			`	"We can give you a power generator that's as big as two fighters and a lot of cooling outfits to support it.`
```

Reply:
Applied manually while fixing another error on this line.